website/src/main/tut/docs/rules/Disable.md

```diff
@@ -29,4 +117,3 @@ println(
 scalafix.website.rule("Disable", DisableConfig.default)
```

Comment:
This will print out the latest version of the config, we only want to copy paste the old settings.